classes/install/Upgrade.inc.php

```diff
@@ -1585,17 +1594,17 @@ function convertCommentsToEditor() {
 				
 				$queryDao->insertObject($query);
 				$queryDao->resequence(ASSOC_TYPE_SUBMISSION, $row['article_id']);
```

Comment:
ah, it should. That was one of the typos I was meaning to fix, but apparently did not succeed. I will commit in a while...